config/initializers/urls.rb

```diff
@@ -43,6 +43,6 @@
 
 STATUS_PAGE_URL = ENV.fetch("STATUS_PAGE_URL", "https://status.demarches-simplifiees.fr")
 DEMANDE_INSCRIPTION_ADMIN_PAGE_URL = ENV.fetch("DEMANDE_INSCRIPTION_ADMIN_PAGE_URL", "https://www.demarches-simplifiees.fr/commencer/demande-d-inscription-a-demarches-simplifiees")
-MATOMO_IFRAME_URL = "https://stats.data.gouv.fr/index.php?module=CoreAdminHome&action=optOut&language=fr&&fontColor=333333&fontSize=16px&fontFamily=Muli"
+MATOMO_IFRAME_URL = ENV.fetch("MATOMO_IFRAME_URL", "https://#{ENV.fetch('MATOMO_HOST', 'stats.data.gouv.fr')}/index.php?module=CoreAdminHome&action=optOut&language=fr&&fontColor=333333&fontSize=16px&fontFamily=Muli")
```

Comment:
Si on rajoute `MATOMO_IFRAME_URL` dans `env.example`, alors on sait qu'elle fait partie des variables d'env obligatoires, et donc qu'elle sera forcément définie dans l'environnement au runtime.

Du coup je serais pour supprimer la valeur par défaut ici (et mettre une valeur vide si Matomo est désactivé).

```suggestion
MATOMO_IFRAME_URL = ENV.fetch("MATOMO_ENABLED") == "enabled" ? ENV.fetch("MATOMO_IFRAME_URL") : ""
```

Reply:
En effet. Cela dit, la valeur par défaut contient une URL aux nombreux paramètres assez génériques mais difficilement prévisibles. L'idée ici était plutôt de proposer des paramètres par défaut, mais surchargeables au besoin. Ainsi, j'aurais plutôt vu cette constante dans `env.example.optional`. Ce qui évite de forcer la définition d'une URL plutôt complexe. Qu'en penses-tu ?

Reply:
Oui, tu as raison, ça m'a l'air bien de la mettre dans `env.example.optional` 👍